deploy/scripts/src/cri-lime/test.ts

```diff
@@ -435,16 +430,47 @@ export function drivingLicence (): void {
         consentDVACheckbox: 'true'
       }
 
-  group(groups[2], () => { // B02_Driving_03_${licenceIssuer}_EnterDetailsConfirm
-    timeRequest(() => {
-      res = group(groups[3].split('::')[1], () => timeRequest(() => // 01_CRICall
-        res.submitForm({ fields, params: { redirects: 2 }, submitSelector: '#continue' }), { isStatusCode302 }))
-      res = group(groups[4].split('::')[1], () => timeRequest(() => // 02_CoreStubCall
-        http.get(res.headers.Location, { headers: { Authorization: `Basic ${encodedCredentials}` } }),
-      { isStatusCode200, ...pageContentCheck('Verifiable Credentials') }))
-    }, {})
-  })
-  iterationsCompleted.add(1)
+  if (licenceIssuer === 'DVA') {
+    res = group(groups[1], () => timeRequest(() => // B02_Driving_02_Select_DVA
+      res.submitForm({
+        fields: { licenceIssuer },
+        submitSelector: '#submitButton'
+      }),
+    { isStatusCode200, ...pageContentCheck('Enter your details exactly as they appear on your UK driving licence') }))
+
+    sleepBetween(1, 3)
+
+    group(groups[2], () => { // B02_Driving_03_EnterDetailsConfirm_DVA
+      timeRequest(() => {
+        res = group(groups[3].split('::')[1], () => timeRequest(() => // 01_CRICall
+          res.submitForm({ fields, params: { redirects: 2 }, submitSelector: '#continue' }), { isStatusCode302 }))
+        res = group(groups[4].split('::')[1], () => timeRequest(() => // 02_CoreStubCall
+          http.get(res.headers.Location, { headers: { Authorization: `Basic ${encodedCredentials}` } }),
+        { isStatusCode200, ...pageContentCheck('Verifiable Credentials') }))
+      }, {})
+    })
+    iterationsCompleted.add(1)
+  } else if (licenceIssuer === 'DVLA') {
+    res = group(groups[5], () => timeRequest(() => // B02_Driving_02_Select_DVLA
+      res.submitForm({
+        fields: { licenceIssuer },
+        submitSelector: '#submitButton'
+      }),
+    { isStatusCode200, ...pageContentCheck('Enter your details exactly as they appear on your UK driving licence') }))
+
+    sleepBetween(1, 3)
+
+    group(groups[6], () => { // B02_Driving_03_EnterDetailsConfirm_DVLA
+      timeRequest(() => {
+        res = group(groups[7].split('::')[1], () => timeRequest(() => // 01_CRICall
+          res.submitForm({ fields, params: { redirects: 2 }, submitSelector: '#continue' }), { isStatusCode302 }))
+        res = group(groups[8].split('::')[1], () => timeRequest(() => // 02_CoreStubCall
+          http.get(res.headers.Location, { headers: { Authorization: `Basic ${encodedCredentials}` } }),
+        { isStatusCode200, ...pageContentCheck('Verifiable Credentials') }))
+      }, {})
+    })
+    iterationsCompleted.add(1)
+  }
```

Comment:
```suggestion
  } else if (licenceIssuer === 'DVLA') {
    res = group(groups[5], () => timeRequest(() => // B02_Driving_02_Select_DVLA
      res.submitForm({
        fields: { licenceIssuer },
        submitSelector: '#submitButton'
      }),
    { isStatusCode200, ...pageContentCheck('Enter your details exactly as they appear on your UK driving licence') }))

    sleepBetween(1, 3)

    group(groups[6], () => { // B02_Driving_03_EnterDetailsConfirm_DVLA
      timeRequest(() => {
        res = group(groups[7].split('::')[1], () => timeRequest(() => // 01_CRICall
          res.submitForm({ fields, params: { redirects: 2 }, submitSelector: '#continue' }), { isStatusCode302 }))
        res = group(groups[8].split('::')[1], () => timeRequest(() => // 02_CoreStubCall
          http.get(res.headers.Location, { headers: { Authorization: `Basic ${encodedCredentials}` } }),
        { isStatusCode200, ...pageContentCheck('Verifiable Credentials') }))
      }, {})
    })
  }
  iterationsCompleted.add(1)
```
Keep `iterationsCompleted` as only called once, at the end of the scenario

Reply:
Updated